packages/ui/src/theme/css/component/authenticator.scss

```diff
@@ -37,16 +37,25 @@
 [data-amplify-authenticator] [data-amplify-container] {
   /* Fix z-index of texture being over the Authenticator */
   position: relative;
-
-  background-color: var(--amplify-colors-background-primary);
-  box-shadow: var(--amplify-shadows-small);
   place-self: center;
   width: 30rem;
   @media (max-width: 30rem) {
     width: 100%;
   }
 }
 
+[data-amplify-authenticator] [data-amplify-container] > {
+  [data-amplify-header],
+  [data-amplify-footer] {
+    text-align: center;
```

Comment:
Opinionated CSS here, we center the top-level header and footers (not sign-in-header, sign-up-header, etc!) here. This can be overridden by customers, or just be removed if we think it's unnecessary.

Reply:
Nice!